nvflare/apis/utils/fl_context_utils.py

```diff
@@ -36,6 +36,7 @@ def get_serializable_data(fl_ctx: FLContext):
 
 
 def generate_log_message(fl_ctx: FLContext, msg: str):
+    _identity_ = "identity"
```

Comment:
Those are constants. They should be all caps and placed at module level.

Reply:
These are only used here locally in the generate_log_message(). I think it's okay to be here. And they are all consistent with all the others.